pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1573,6 +1576,52 @@ func registerCDC(r registry.Registry) {
 			ct.waitForWorkload()
 		},
 	})
+
+	r.Add(registry.TestSpec{
+		Name:             "cdc/kafka-chaos-integrity",
+		Owner:            `cdc`,
+		Cluster:          r.MakeClusterSpec(6, spec.CPU(16)),
+		Leases:           registry.MetamorphicLeases,
+		CompatibleClouds: registry.AllExceptAWS,
+		Suites:           registry.Suites(registry.Nightly),
```

Comment:
TODO: change this to manual only for now since it fails with the sarama kafka sink